### PR TITLE
Tooltips on comparative charts

### PR DESCRIPTION
**Pull request by magopian**
[user story](https://www.notion.so/Info-bulles-graphe-R-sultat-000634b5c7de4d3c902d1f1557b124b0)

### CODE REVIEW

File: src/Page/Food/Builder.elm [hunk 1]
```diff
@@ -91,6 +94,7 @@ type Msg
     | DeletePreparation Preparation.Id
     | LoadQuery Query
     | NoOp
+    | OnHover (List (CI.Many ComparativeChart.Entry CI.Any))
```
Comment (n1k0) [inline src/Page/Food/Builder.elm:97]: Maybe name this msg slightly more explicitely eg. `OnChartHover` or `OnComparativeChartHover`?

File: src/Page/Food/Builder.elm [hunk 2]
```diff
@@ -131,6 +135,7 @@ init ({ db, builderDb, queries } as session) trigram maybeQuery =
               , comparisonUnit = ComparatorView.PerKgOfProduct
               , displayChoice = ComparatorView.IndividualImpacts
               , modal = NoModal
+              , hovering = []
```
Comment (n1k0) [inline src/Page/Food/Builder.elm:138]: Same, maybe `chartHovering` or `comparativeChartHovering`?

**Review by n1k0 — COMMENTED** [review]
Looks nice, is it possible to decrease the font size and round the numbers though?

**Comment by n1k0** [comment]
Also just wondering if there's any available API in elm-chart to have hovering applied on individual bar segments rather than on the whole segment itself?

File: src/Page/Food/Builder.elm (hunk 2 above)
Comment (magopian) [reply src/Page/Food/Builder.elm:137]: Renamed to `chartHovering`

**Review by n1k0 — APPROVED** [review]
The patch does the job, good work. We may want to investigate switching this chart to use Highcharts, but in another PR, later, if needed.

**Comment by n1k0** [comment]
Hmm just tested the review app, we have this:
image

Is that expected?

**Comment by magopian** [comment]
Yep, it's what we settled down to in the notion card ;)

**Pull request merged**